src/firmware/boards/robot_stm32h7/io/vision.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+// TODO (#2083) implement this stuff
```

Comment:
Could we add jdocs to at least specify return type units? One jdoc per-group should be sufficient.

Reply:
added java docs and moved the TODO's to the c file